vue-app/src/components/ProjectListItem.vue

```diff
@@ -35,34 +35,7 @@
           Register
         </button>
 
-        <form action="#" v-if="shouldShowCartInput">
-          <!-- TODO: if a user is unconnected, adding to cart should trigger wallet connection -->
-          <div class="input-button">
-              <img style="margin-left: 0.5rem;" height="24px" v-if="!inCart" src="@/assets/dai.svg">
-              <input
-                v-model="amount"
-                class="input"
-                name="amount"
-                :placeholder="defaultContributionAmount"
-                autocomplete="on"
-                onfocus="this.value=''"
-                v-if="!inCart"
-              >
-              <input type="submit"
-                v-if="hasContributeBtn() && !inCart"
-                class="donate-btn"
-                :disabled="!canContribute()"
-                @click="contribute()"
-                value="Add to cart"
-              >
-              <div 
-                v-if="hasContributeBtn() && inCart"
-                class="donate-btn-full"
-              >
-              In cart 🎉
-              </div>
-          </div>
-        </form>
+        <add-to-cart-button v-if="shouldShowCartInput" :project="project" />
 
         <!-- <button
```

Comment:
Seems this code commented out can be deleted.

Reply:
As a result I believe some logic within here can also be removed (e.g. `hasContributeBtn`)

Reply:
Cool, yes. Fewer lines of code! 🎉